src/components/RenderRouter/GoContentItem.scss

```diff
@@ -94,6 +109,21 @@
     .GoContentClear{
         clear:left;
     }
+
+    .GoContentItemA {
+        color: black;
+        border-bottom: 1px solid black;
+        padding-bottom: 1px;
+    }
+
+    .GoContentItemA:hover {
```

Comment:
Since you want the link to look the same I would recommend
```
    .GoContentItemA {
        color: black;
        border-bottom: 1px solid black;
        padding-bottom: 1px;

        &:hover {
            color: inherit;
            text-decoration: inherit;
        }
    }
```
This makes it clear that you're not adding new styles, you're just trying to "reset" them. This could be used throughout this PR.